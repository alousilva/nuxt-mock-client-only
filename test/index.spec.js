import { shallowMount } from '@vue/test-utils'
import Index from '@/pages'

describe('Index', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(Index)
    expect(wrapper.element).toMatchSnapshot()
  })
})
