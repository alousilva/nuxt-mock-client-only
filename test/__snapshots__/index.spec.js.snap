// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should match the snapshot 1`] = `
<div
  class="container"
>
  <div>
    <h1
      class="title"
    >
      
      nuxt-mock-client-only
    
    </h1>
     
    <div
      class="components"
    >
      <componenta-stub />
       
      <div>
        <componentb-stub />
      </div>
    </div>
  </div>
</div>
`;
